swift_scality_backend/diskfile.py

```diff
@@ -47,26 +48,28 @@ class DiskFileWriter(object):
     requests. Serves as the context manager object for DiskFile's create()
     method.
 
-    :param filesystem: internal file system object to use
```

Comment:
No docstring to describe the client_collection param. I know it's self explanatory but, it's always nice to have all param taken into account.